lib/trino-parquet/src/main/java/io/trino/parquet/reader/flat/DefinitionLevelDecoder.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.parquet.reader.flat;
+
+public interface DefinitionLevelDecoder
```

Comment:
Shouldn't this be called NullDecoder or something like that?

Reply:
It is a definition levels decoder for the special cases where definition level is 0 or 1. I've renamed this interface to FlatDefinitionLevelDecoder and renamed the implementation to NullsDecoder.